packages/dialtone-vue3/prototypes/dialpad/leftbar/components/leftbar_section.vue

```diff
@@ -0,0 +1,64 @@
+<template>
+  <dt-collapsible
+    class="leftbar-section"
+    :open="isOpen"
+  >
+    <template #anchor="{ attrs }">
+      <div class="leftbar-section-header__wrapper">
+        <dt-button
+          v-bind="attrs"
+          class="d-bar-pill d-w100p d-lh12 d-headline--sm d-fc-tertiary d-us-none"
+          label-class="d-jc-flex-start d-w100p"
+          importance="clear"
+          kind="muted"
+          @click="defaultToggleOpen"
+        >
+          <dt-icon
+            :name=" isOpen ? 'chevron-down' : 'chevron-right'"
+            size="300"
+          />
+          <span class="d-truncate d-w128 d-ta-left">{{ title }}</span>
```

Comment:
Had to put this d-w128 to achieve truncate, we'll need to make this with JS otherwise if we need it to be dynamically updated as the slot can be anything.

Reply:
I would put this on a `<!-- -->` in the template, kind of a to do or just to let it know this what you are saying and not missing it in the PR.

Reply:
does max-width work better? or it needs to be fixed?

Reply:
Yes! max-width worked better indeed. 

I would like a design review here, is this ok to do? or should we set a fixed percent of it for the channel name and the rest for the button?

Something like 60/40 instead of a fixed 12.8 rem? I think the problem is that on product the leftbar is resizable.

![Screenshot 2024-06-20 at 12 07 21 p m](https://github.com/dialpad/dialtone/assets/87546543/1fa6f152-4839-4f2b-91b9-bb339e5df2c4)
![Screenshot 2024-06-20 at 12 07 42 p m](https://github.com/dialpad/dialtone/assets/87546543/0d792cac-7dd0-4d9c-935b-723eb047dd43)

@francisrupert @fede-dp